ios/Exponent/Versioned/Core/UniversalModules/EXScopedFilePermissionModule.m

```diff
@@ -0,0 +1,12 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+#import "EXScopedFilePermissionModule.h"
+#import <EXFileSystemInterface/EXFileSystemInterface.h>
+
+@implementation EXScopedFilePermissionModule
+
+- (EXFileSystemPermissionFlags)getPathPermissionsForExternalDircetories:(NSString *)path
+{
+  return EXFileSystemPermissionNone;
```

Comment:
If https://github.com/expo/expo/pull/2841 is merged, would you care to upgrade this implementation to something similar to https://github.com/expo/expo/pull/2841/files#diff-d94ddcbfd88f8b539a2529d22f3fcdcd? 🙂